src/LandingPage.tsx

```diff
@@ -65,31 +67,51 @@ const SignIn = styled.div`
 `;
 
 const LandingPage: React.FC = () => {
+  const auth = useContext(AuthContext);
+
   function loginSuccessHandler(
-    response: GoogleLoginResponse | GoogleLoginResponseOffline
+    googleResponse: GoogleLoginResponse | GoogleLoginResponseOffline
   ) {
-    console.log(response);
-    let onlineResponse = response as GoogleLoginResponse;
+    let onlineResponse = googleResponse as GoogleLoginResponse;
     if (!onlineResponse) {
       // Not supported yet
       return;
     }
-    // TODO: Send onlineResponse.tokenId to the backend to validate the token, get the user object, and store that in localStorage
-    localStorage.setItem("token", onlineResponse.tokenId);
-    localStorage.setItem("user", onlineResponse.profileObj.name);
+
+    const options = {
+      credentials: "include" as const,
```

Comment:
What's the `as const` doing?

Reply:
In short, satisfying typescript. When putting in the options directly into the fetch method, TS figures out that "include" is one of "include" | "same-origin" | "omit"
But, when defining the options separately, TS thinks that it's just a generic string and complains.

Reply:
Ah, I suspected something of the sort :P